src/main/core/cell/cell-handler.component.ts

```diff
@@ -120,7 +122,7 @@ export class CellHandlerComponent implements OnInit, AfterViewInit {
 	}
 
 	startBatchEdit(e) {
-		this.root.table.view.addClass('q-grid-noselect');
+		this.root.model.selection({mode: 'range'});
```

Comment:
you need to persist selectionMode every time on batchEdit,
the same for editMode